src/crypto/aes_256.c

```diff
@@ -0,0 +1,25 @@
+#include "crypto.h"
+
+static const CipherGetter chiphers[] = {
+    EVP_aes_256_cbc,
+    EVP_aes_256_ecb,
+    EVP_aes_256_cfb8,
+    EVP_aes_256_ofb,
+};
+
+static inline const EVP_CIPHER *get_aes_256_cipher(const CIPHER_MODES mode)
+{
+    return chiphers[mode]();
```

Comment:
Corrected spelling of 'chiphers' to 'ciphers'.

```suggestion
    return ciphers[mode]();
```